src/Storage/Storage.Management/Models/PSDataPolicy.cs

```diff
@@ -122,18 +126,18 @@ public PSManagementPolicyDefinition()
         {
         }
 
-        public PSManagementPolicyDefinition(ManagementPolicyDefinition defination)
+        public PSManagementPolicyDefinition(Track2.Models.ManagementPolicyDefinition defination)
         {
             this.Actions = defination.Actions is null ? null : new PSManagementPolicyActionGroup(defination.Actions);
             this.Filters = defination.Filters is null ? null : new PSManagementPolicyRuleFilter(defination.Filters);
         }
-        public ManagementPolicyDefinition ParseManagementPolicyDefination()
+        public Track2.Models.ManagementPolicyDefinition ParseManagementPolicyDefination()
         {
-            return new ManagementPolicyDefinition()
-            {
-                Actions = this.Actions is null ? null : this.Actions.ParseManagementPolicyAction(),
-                Filters = this.Filters is null ? null : this.Filters.ParseManagementPolicyFilter()
-            };
+            Track2.Models.ManagementPolicyAction actions = this.Actions is null ? null : this.Actions.ParseManagementPolicyAction();
```

Comment:
Try the syntactic sugar: `actions = this.Actions?.ParseManagementPolicyAction();`

Reply:
Done